File: 20260704/exos/01_calcul.py
# ============================================================
# Project Zomboid — stock de conserve
# ============================================================
# Tu as 12 boîtes de soupe, tu en manges 4, puis tu en trouves 7
# en fouillant une maison. Combien en reste-t-il ?
#
# Lance : python3 01_calcul.py
# ============================================================

conserves = 12 - 4 + 7

# Calcule le stock final en une expression.
#
# 12 - 4 = 8   (après avoir mangé)
# 8 + 7 = 15   (après la fouille)
#
# Résultat attendu : conserves == 15
#
# Indice : une expression avec - et +

# À toi :


# --- Vérification (ne pas modifier) ---
assert conserves == 15, "Recompte : 12 - 4 + 7 = ?"
print("✅ Correct !")

File: 20260718/exos/01_calcul.lua
-- ============================================================
-- Project Zomboid — stock de bandages
-- ============================================================
-- Tu as 14 bandages. Tu en utilises 5 sur des blessures,
-- puis tu en trouves 3 dans une pharmacie. Combien en reste-t-il ?
--
-- Lance : lua 01_calcul.lua
-- ============================================================

bandages = 0

-- Calcule les bandages restants avec une seule expression.
--
-- 14 - 5 = 9   (après les soins)
-- 9 + 3 = 12   (après la pharmacie)
--
-- Résultat attendu : bandages == 12
--
-- Indice : une expression avec - et +

-- À toi :
bandages = 14 - 5 + 3

-- --- Vérification (ne pas modifier) ---
assert(bandages == 12, "Recompte : 14 - 5 + 3 = ?")
print("✅ Correct !")
